examples/room-manager/src/routes/rooms.ts

```diff
@@ -0,0 +1,42 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { ServerMessage } from '@fishjam-cloud/js-server-sdk/proto';
+
+import { parseError } from '../errors';
+import { fishjamPlugin } from '../plugins';
+import { GetPeerAccessQueryParams, startRecordingSchema, queryStringPeerEndpointSchema } from '../schema';
+import { httpToWebsocket, removeTrailingSlash } from '../utils';
+
+export async function rooms(fastify: FastifyInstance) {
+  await fastify.register(fishjamPlugin);
+
+  const getRoomAccessHandler = async (roomName: string, peerName: string, res: FastifyReply) => {
+    try {
+      const accessData = await fastify.fishjam.getPeerAccess(roomName, peerName);
+      const url = httpToWebsocket(fastify.config.FISHJAM_URL);
+
+      // When creating a URL object from a URL without a path (e.g., `http://localhost:5002`),
+      // the `href` field may contain an additional '/' at the end (`http://localhost:5002/`).
+      const urlWithoutTrailingSlash = removeTrailingSlash(url);
+
+      return { ...accessData, url: urlWithoutTrailingSlash };
```

Comment:
**question:** Does it work without `res.send`? Is there some kind of middleware?

Reply:
yes